### PR TITLE
BeyondTrust Password Safe XSIAM Mapping

### PR DESCRIPTION
**Pull request by cweltPA**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-9669](https://jira-dc.paloaltonetworks.com/browse/CIAC-9669)

## Description
This PR contains Modeling & Parsing Rules for Beyond Trust Password Safe PBPS (Password-Broker Password Safe) & AppAudit event logs. 

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/BeyondTrust_Password_Safe/README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
 # Beyond Trust Password Safe
-Unified password and session management for seamless accountability and control over privileged accounts.
\ No newline at end of file
+Unified password and session management for seamless accountability and control over privileged accounts.
+
+<~XSIAM>
+
+## Cortex XSIAM SIEM Content
+
+This pack includes Cortex XSIAM SIEM content for parsing and modeling the syslog events that are forwarded from BeyondTrust Beyond Safe.
```
Comment (ShirleyDenkberg) [inline Packs/BeyondTrust_Password_Safe/README.md:8]: Should this be BeyondTrust **Password** Safe?

File: Packs/BeyondTrust_Password_Safe/README.md [hunk 2]
```diff
@@ -1,2 +1,88 @@
...
+1. In BeyondInsight, go to *Configuration* &rarr; *General* &rarr; *Connectors*.
+2. From the *Connectors* pane, click **Create New Connector**.
+3. Enter a name for the connector, for e.g., "*Cortex XSIAM*". 
+4. Select **Syslog Event Forwarder** under the *Connector Type* list.
+5. Click **Create Connector** to open the *Syslog Event Forwarder* pane.
+6. Leave **Active (yes)** enabled.
+7. Provide the required details of the target Cortex XSIAM Broker VM syslog server:
+   - **`Available Output Pipelines`** - Select the requested transmission protocol for forwarding the syslog messages: *TCP*, *TCP-SSL*, or *UDP*.
+   - **`Host Name`** - Enter the IP address or hostname of the target Cortex XSIAM Broker VM syslog server.
+   - **`Port`** - enter the port number that the target Broker VM Syslog service is listening on for receiving syslog message from BeyondTrust Password Safe.
```
Comment (ShirleyDenkberg) [inline Packs/BeyondTrust_Password_Safe/README.md:27]:
```suggestion
   - *`Available Output Pipelines`* - Select the requested transmission protocol for forwarding the syslog messages: *TCP*, *TCP-SSL*, or *UDP*.
   - *`Host Name`* - Enter the IP address or hostname of the target Cortex XSIAM Broker VM syslog server.
   - *`Port`* - enter the port number that the target Broker VM Syslog service is listening on for receiving syslog message from BeyondTrust Password Safe.
```

File: Packs/BeyondTrust_Password_Safe/README.md [hunk 3]
```diff
@@ -1,2 +1,88 @@
...
+8. Select one of the following output formats: *Comma Delimited* or *Tab Delimited*. Other formats are currently unsupported.
+9. Select an optional syslog **Facility** from the list.
```
Comment (ShirleyDenkberg) [inline Packs/BeyondTrust_Password_Safe/README.md:29]:
```suggestion
9. Select an optional syslog *Facility* from the list.
```

File: Packs/BeyondTrust_Password_Safe/README.md [hunk 4]
```diff
@@ -1,2 +1,88 @@
...
+10. Select **Format Specification**.
+11. Select the events that you want to forward to Cortex XSIAM.
+12. Click **Test Connector** to determine if the event forwarding configuration is successful.
+13. Click **Create Connector**.
+
+See BeyondTrust Password Safe [Enable Syslog Event Forwarding](https://www.beyondtrust.com/docs/beyondinsight-password-safe/bi/integrations/third-party/snmp-trap-and-syslog.htm#:~:text=Enable%20Syslog%20Event%20Forwarding) guide for additional details. Remark: The timestamp extracted from the BeyondTrust Beyond Safe events are interpreted in UTC timezone.
```
Comment (ShirleyDenkberg) [inline Packs/BeyondTrust_Password_Safe/README.md:35]: Should this be BeyondTrust **Password** Safe?

File: Packs/BeyondTrust_Password_Safe/ParsingRules/BeyondTrust_Password_Safe/BeyondTrust_Password_Safe.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+id: BeyondTrust_PasswordSafe_ParsingRule
```
Comment (ShirleyDenkberg) [inline Packs/BeyondTrust_Password_Safe/ParsingRules/BeyondTrust_Password_Safe/BeyondTrust_Password_Safe.yml:1]: This ID is written differently than the one for modeling rules
Modeling: BeyondTrust_Password_Safe_ModelingRule
Parsing: BeyondTrust_PasswordSafe_ParsingRule (vs BeyondTrust_Password_Safe_ParsingRule

File: Packs/BeyondTrust_Password_Safe/README.md [hunk 6]
```diff
@@ -1,2 +1,88 @@
...
+
+In order to use the collector, use the [Broker VM](#broker-vm) option.
+
+#### Broker VM
+You will need to use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Configure-the-Broker-VM).
+
+You can configure the specific vendor and product for this instance.
+
+1. Navigate to **Settings** &rarr; **Configuration** &rarr; **Data Broker** &rarr; **Broker VMs**. 
+2. Go to the **APPS** column under the **Brokers** tab and add the **Syslog** app for the relevant broker instance. If the **Syslog** app already exists, hover over it and click **Configure**.
```
Comment (ShirleyDenkberg) [inline Packs/BeyondTrust_Password_Safe/README.md:49]:
```suggestion
2. Go to the **APPS** column under the **Brokers** tab and add the **Syslog** app for the relevant broker instance. If the Syslog app already exists, hover over it and click **Configure**.
```

**Comment by ShirleyDenkberg** [comment]
@OBavly Doc review completed.

File: Packs/BeyondTrust_Password_Safe/README.md (hunk 1 above)
Comment (cweltPA) [reply Packs/BeyondTrust_Password_Safe/README.md:8]: I agree, BeyondTrust seems correct in contrast to Beyond Trust, I'll fix it accordingly.

**Review by OBavly — APPROVED** [review]

**Pull request merged**